dashboard/src/client.ts

```diff
@@ -48,6 +48,18 @@ function connectIngestMonitor() {
       handleIngestEvent(data.ingestValue);
     }
   };
+  socket.onclose = () => {
+    console.error("Ingest monitor socket closed");
+    setTimeout(() => {
+      console.log("Reconnecting ingest monitor socket...");
+      client.ingest.ingestMonitorRequest().then(() => {
```

Comment:
I'd consider putting this initial request inside the `connectIngestMonitor` function directly, and the same for storage. It would reduce some duplication, leaving the calls in App.vue just with `client.connectIngestMonitor();` and `client.connectStorageMonitor();` inside the if, and similar here on close.

https://github.com/artefactual-sdps/enduro/blob/main/dashboard/src/App.vue#L19-L24

Reply:
@jraddaoui I don't understand what you mean.  The `connectIngestMonitor` is autogenerated, I'm not sure how we would modify the function put the `ingestMonitorRequest()` call "inside" it. :confused:

Reply:
Oh, I see `connectIngestMonitor()` isn't auto-generated, `ingestMonitorRequest()` is auto-generated.